kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -245,6 +281,75 @@
         const newMsg = createReturnMsg({ message, data: __version });
         return this.hashi.mediator.sendMessage(newMsg);
       },
+      sendChannelMetadata(message) {
+        return ChannelResource.fetchModel({ id: this.topic.channel_id })
```

Comment:
We can retrieve this from bootstrapped metadata now, see this composable: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/composables/useChannels.js

This will avoid a request.

Reply:
I've just updated this API call to use the `useChannels` function instead of making a request to the backend.